specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -814,6 +814,78 @@
         }
       }
     },
+    "AcsRouterWorkerUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterWorkerUpdated event",
+      "type": "object",
+      "properties": {
+        "workerId": {
+          "description": "Router Worker Updated Worker Id",
+          "type": "string"
+        },
+        "queueAssignments": {
+          "description": "Router Worker Updated Queue Info",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterQueueDetails"
+          }
+        },
+        "channelConfigurations": {
+          "description": "Router Worker Updated Channel Configuration",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcsRouterChannelConfiguration"
+          }
+        },
+        "totalCapacity": {
+          "description": "Router Worker Updated Total Capacity",
+          "type": "integer",
+          "format": "int32"
+        },
+        "labels": {
+          "type": "object",
+          "description": "Router Worker Updated Labels",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tags": {
+          "type": "object",
+          "description": "Router Worker Updated Tags",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "updatedWorkerProperties": {
+          "description": "Router Worker Properties Updated",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "An individual property updated in the Router Worker",
+            "enum": [
+              "AvailableForOffers",
+              "TotalCapacity",
+              "QueueAssignments",
+              "Labels",
+              "Tags",
+              "ChannelConfigurations",
+              "MaxConcurrentOffers"
+            ],
+            "x-ms-enum": {
+              "name": "AcsRouterUpdatedWorkerProperty",
+              "modelAsString": true
+            }
+          }
+        },
+        "x-ms-examples": {
```

Comment:
These are nested under properties so it is cause errors.

Reply:
nice catch. updated as well